lib/modules/keyboardNav.js

```diff
@@ -1207,6 +1215,16 @@ function reply() {
 	}
 }
 
+function showChildComments() {
+	const selected = SelectedEntry.selectedThing;
+	if (!selected) return;
+
+	const button = selected.entry.querySelector('a.toggleChildren');
```

Comment:
Instead of clicking the button, could you call the "toggle children on this entry" method directly?